source/form.html

```diff
@@ -47,121 +47,124 @@
       </nav>
     </div>
   </header>
-  <main class="page-form">
+  <main class="page-main">
     <h1 class="visually-hidden">Форма регистрации</h1>
-    <span class="page-form__announce">Конкурс на самое
+    <p class="page-form__announce">Конкурс на самое
       безбашенное
-      путешествие с Пинк</span>
-    <form class="page-form__registration-form">
+      путешествие с Пинк</p>
+    <form class="page-main__registration" action="https://echo.htmlacademy.ru/" method="post">
       <p>Поделитесь своей историей и получите
         шанс выиграть ценный приз — 1000
         миль на вашу бонусную карту!</p>
-      <div class="registration-form__name-container">
-        <label for="surname-field" class="registration-form__label">
-          ФАМИЛИЯ:
-        </label>
-        <input class="field" placeholder="Укажите фамилию *" type="text" id="surname-field" name="surname-field"
-          required>
-        <label for="name-field" class="registration-form__label">
-          ИМЯ:
-        </label>
-        <input class="field" placeholder="Введите ваше имя *" type="text" id="name-field" name="name-field" required>
-        <label for="second-name-field" class="registration-form__label">
-          ОТЧЕСТВО:
-        </label>
-        <input class="field" placeholder="Ну и отчество тоже" type="text" id="second-name-field"
-          name="second-name-field">
+      <div class="registration__name-form">
+        <div class="field">
+          <label for="surname" class="registration__label">
+            Фамилия:
+          </label>
+          <input class="field__input" placeholder="Укажите фамилию *" type="text" id="surname" name="surname" required>
+        </div>
+        <div class="field">
+          <label for="name" class="registration__label">
```

Comment:
Не логичнее ли внутри блока делать оба элемента принадлежащим именно ему?
```suggestion
          <label for="name" class="field__label">
```